CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- [#25] API to set log level for a dogu
```

Comment:
Currently this is API does a bit more: setting a dogu's log level and conditionally restart it (which brings possibly unwanted downtime). I'd expect to find this tidbit in a change log.